server/src/utils/request.ts

```diff
@@ -0,0 +1,101 @@
+import axios from 'axios';
+import FormData from 'form-data';
+import logger from '@shared/Logger';
+
+export async function post(endpoint_url: string, data: FormData, extra: any) {
+    return await request({ 
+        method: 'post',
+        url: endpoint_url,
+        data: data,
+        headers: extra}, true);
+}
+
+export async function get(endpoint_url: string, req_qry: any, token: string) {
+    return await request({ 
+        method: 'get',
+        url: endpoint_url,
+        params: req_qry,
+        headers: {
+            'Authorization': `Bearer ${token}`
+        }}, true);
+}
+
+export async function request(config: any, retry_flag: boolean) {
+    var resp = null

```

Comment:
```suggestion
    let resp = null

```
In modern JS it's common to use either `let` if the variable will change over time or `const` if it will never change.